documentation-site/components/thumbs/template.js

```diff
@@ -24,6 +24,7 @@ function template(api, opts, values) {
   return template.ast`
 ${header}
 ${imports}
+${'\n'}
```

Comment:
I added this line break to satisfy eslint rules `import/newline-after-import`
![Uploading image.png…]()